Content.Shared/Actions/ConfirmableActionComponent.cs

```diff
@@ -0,0 +1,45 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared.Actions;
+
+/// <summary>
+/// An action that must be confirmed before using it.
+/// Using it for the first time primes it, after a delay you can then confirm it.
+/// Used for dangerous actions that cannot be undone (unlike screaming).
+/// </summary>
+[RegisterComponent, NetworkedComponent, Access(typeof(ConfirmableActionSystem))]
+public sealed partial class ConfirmableActionComponent : Component
+{
```

Comment:
Are the fields supposed to be networked.

Reply:
they shouldnt need to be since everything is in shared

only using NetworkedComponent because another component does

Reply:
@metalgearsloth do i need to change anything